shared/people/index.native.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react'
+import {
+  Avatar,
+  NativeSafeAreaView,
+  NativeRefreshControl,
+  NativeStyleSheet,
+  ScrollView,
+} from '../common-adapters/index.native'
+import {PeoplePageSearchBar, PeoplePageList} from './index.shared'
+import {type Props} from '.'
+import {globalColors, globalStyles} from '../styles'
+
+const People = (props: Props) => (
```

Comment:
might be better to just use Listview so this and desktop can be the same

Reply:
do you mean `common-adapters/list`?

Reply:
yea

Reply:
talked over IRL: there's enough different between native and desktop (native-only components etc.) to cleanly try to combine these two versions. This separation could definitely be better (and will have to be to transition to a virtualized list for this).

(does this sound ok to you @chrisnojima? I may be forgetting something)